src/assets/styles/header.scss

```diff
@@ -68,7 +68,7 @@
 
 @keyframes slideUp {
   0% { top: 105vh; }
-  100% { top: 0vh; }
+  100% { top: 4.5em; }
```

Comment:
(Setting it in a fixed way like this is fairly brittle because if the styles in the header / logo area change it could break this too -- this is why it was calculating the top position via JS. But this should be fine, seems to look good across browsers / different zoom levels / etc. and I doubt there are going to be major changes to those other styles....probably my previous solution was overengineered ;) )